addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss

```diff
@@ -50,9 +51,14 @@
         cursor: pointer;
         background-color: var(--o-table-ui-bg, #{$o-white});
         color: var(--o-table-ui-color, #{$o-main-text-color});
-        border: $border-width solid var(--o-table-ui-border, #{$border-color});
-        border-radius: 5px;
-        padding: 2px 3px;
+        border: $border-width solid rgba($color: $o-enterprise-color, $alpha: 1.0);
+        width: 100%;
+        height: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        color: #fff;
+        background-color: rgba($color: $o-enterprise-color, $alpha: 0.7);
```

Comment:
The color is quite surprising in community because in community the main color is blueish rather than purpleish. I think you should replace all the two uses of `$o-enterprise-color` here by another variable. I don't know exactly what variable that is, but I suspect there is some variable that changes color depending on whether you are using community or enterprise. I suspect it is something like primary brand color ?

Reply:
Found it the color was `$o-brand-odoo` I have update it and tested on both community and enterprise.

Reply:
Great, thanks ! 👍